tests/tosi_benchmark_gmg_newton_solver.prm

```diff
@@ -0,0 +1,188 @@
+# Test Newton GMG solver based on
+# benchmarks/newton_solver_benchmark_set/tosi_et_al_2015/input_gmg.prm
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Stokes solver type = block GMG
+    set Number of cheap Stokes solver steps             = 200
+    set Linear solver tolerance                         = 1e-6
+  end
+end
+
+
+# We add the necessary plugins through a shared library
+set Additional shared libraries            = $ASPECT_SOURCE_DIR/benchmarks/tosi_et_al_2015_gcubed/libtosi_benchmark.so
+
+# This is a 2D benchmark.
+set Dimension                              = 2
```

Comment:
same comments as with the other input files, I think most of this can just be included.